docs/src/content/docs/book/contracts.mdx

```diff
@@ -514,13 +514,13 @@ contract Example {
 }
 ```
 
-### Internal functions
+### Member functions {#internal-functions}
```

Comment:
These are private contract functions (or methods). You can define public contract function on the top-level.